ui/app/components/clients/line-chart.js

```diff
@@ -0,0 +1,121 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { max } from 'd3-array';
+// eslint-disable-next-line no-unused-vars
+import { select, selectAll, node } from 'd3-selection';
+import { axisLeft, axisBottom } from 'd3-axis';
+import { scaleLinear, scalePoint } from 'd3-scale';
+import { line } from 'd3-shape';
+import { LIGHT_AND_DARK_BLUE, SVG_DIMENSIONS, formatNumbers } from '../../utils/chart-helpers';
+
+/**
+ * @module LineChart
+ * LineChart components are used to display data in a line plot with accompanying tooltip
+ *
+ * @example
+ * ```js
+ * <LineChart @dataset={dataset} />
+ * ```
+ * @param {array} dataset - dataset is an array of objects
+ */
+
+export default class LineChart extends Component {
```

Comment:
If the charts share some core things, I think you could make a base `Chart` component and then import & extend that so that if shared interactions/styles update, you only have to update it on the base component

Reply:
Can we sync on this to clarify? I think I follow, but want to make sure I'm understanding!